app/controllers/admin/schedules_controller.rb

```diff
@@ -57,17 +77,38 @@ def adapt_params
       params[:schedule][:order_cycle_ids] = params[:order_cycle_ids]
     end
 
-    def check_editable_order_cycle_ids
+    def check_editable_order_cycle_ids_create
+      return unless params[:order_cycle_ids]
+
+      requested = params[:order_cycle_ids]
+      @existing_order_cycle_ids = []
+
+      result = editable_order_cycles(requested)
+
+      params[:order_cycle_ids] = result
```

Comment:
I'd change the method name, it's not just checking, it's changing them. I am not sure what a better name would be.

What about this version?
```ruby
  def define_editable_order_cycle_ids_for_create
      return unless params[:order_cycle_ids]

      @existing_order_cycle_ids = []
      params[:order_cycle_ids] = editable_order_cycles(params[:order_cycle_ids])
  end
```